src/types.ts

```diff
@@ -29,16 +24,25 @@ export interface AutoIncrementOptionsID {
   field?: string;
   /**
    * Set the tracker model name
-   * @default idtracker
+   * @default identitycounter
```

Comment:
why change the default name to this?

Reply:
because the plugin `mongoose-auto-increment` where using this name. and I dont wanna change that.

Reply:
ok